src/services/doppler-legacy-client.ts

```diff
@@ -239,6 +241,8 @@ export function mapHeaderDataJson(json: any) {
       plan: mapPlanEntry(json.user.plan),
     },
     jwtToken: json.jwtToken,
+    notifications: json.notifications,
```

Comment:
Since we are not sure if the input is fine, I think that try to sanitize it is not a bad thing.

```suggestion
    notifications: json.notifications || [],
```